src/main/drivers/accgyro_fake.c

```diff
@@ -134,54 +108,45 @@ bool fakeGyroDetect(gyroDev_t *gyro)
 #ifdef USE_FAKE_ACC
 
 static int16_t fakeAccData[XYZ_AXIS_COUNT];
```

Comment:
I think this should be in `accDev_t` if we want more than one fakeAccDev.
same as `fakeGyroADC[XYZ_AXIS_COUNT]`.